sktime/clustering/evaluation/_plot_clustering.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""Cluster plotting tools"""
+
+__author__ = ["Christopher Holder", "Tony Bagnall"]
+__all__ = ["plot_cluster_algorithm"]
+
+import pandas as pd
+
+from sktime.clustering.base.base_types import Numpy_Or_DF
+from sktime.clustering.base.base import BaseCluster
+from sktime.clustering.partitioning._time_series_k_partition import TimeSeriesKPartition
+from sktime.utils.data_processing import from_nested_to_2d_array
+
+
+def __plot(cluster_values, center, axes):
+    for cluster_series in cluster_values:
+        axes.plot(cluster_series, color="b")
+
+    axes.plot(center, color="r")
+
+
+def plot_cluster_algorithm(
+    model: BaseCluster, predict_series: Numpy_Or_DF, k: int, plt, mpatches=None
+):
+    if isinstance(predict_series, pd.DataFrame):
+        predict_series = from_nested_to_2d_array(predict_series, return_numpy=True)
+    plt.figure(figsize=(5, 10))
```

Comment:
is there a missing import for `matplotlib` or am I not seeing it? Note that matplotlib is a soft dependency, please check the contributing guide to make sure it's handled correctly. Let me know if you need any help with that!